spec.html

```diff
@@ -31108,7 +30972,7 @@
           1. Let _items_ be the List of arguments passed to this function.
           1. Let _C_ be the *this* value.
           1. If IsConstructor(_C_) is *false*, throw a *TypeError* exception.
-          1. Let _newObj_ be ? AllocateTypedArray(_C_, _len_).
+          1. Let _newObj_ be ? Construct(_C_, «_len_»).
```

Comment:
AllocateTypedArray ensures a new TypedArray instance is created, this is not the case with Construct.